src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -45,6 +45,8 @@ RPCHandlerImpl::RPCHandlerImpl(ApplicationManager& app_manager)
     , messages_from_hmi_("AM FromHMI", this)
     , hmi_so_factory_(hmi_apis::HMI_API())
     , mobile_so_factory_(mobile_apis::MOBILE_API())
+    , v4_protocol_so_factory_(
+          ns_smart_device_link_rpc::V1::v4_protocol_v1_2_no_extra())
```

Comment:
@IGapchuk as I could see `hmi_so_factory_`, `mobile_so_factory_`, `v4_protocol_so_factory_` are declared as a class members. I think they could be default-constructed just like this:
```
, hmi_so_factory_()
, mobile_so_factory_()
, v4_protocol_so_factory_()
```
Can't see any reason to create them from the same copy

Reply:
@AKalinich-Luxoft Fixed in 942e36845e9eca9cf6390731be80a8d155e7ecf9.